src/router/chainable_router.rs

```diff
@@ -0,0 +1,71 @@
+use hyper::method::Method;
+use middleware::Middleware;
+use nickel::Nickel;
+use router::http_router::{HttpRouter};
+
+pub struct ChainableRoute<'a, D: Sync + Send + 'static = ()> {
+    path: String,
+    nickel: &'a mut Nickel<D>
```

Comment:
I agree that subrouters are desirable.  I read in one of the other issues that someone was looking to nest routes.  I need to sit down with this for a bit and give it some thought though.